modules/services/trust-relationship/main.tf

```diff
@@ -79,7 +79,7 @@ TEMPLATE
 resource "aws_cloudformation_stack_set_instance" "stackset_instance" {
   count = var.is_organizational ? 1 : 0
 
-  region         = var.region
+  region         = var.region == "" ? null : var.region
```

Comment:
If the default is null, do we even need the `region` input param ? Should we just get rid of it altogether ?